examples/tutorials/tutorial2-python-vector.ipynb

```diff
@@ -0,0 +1,297 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# libCEED for Python examples\n",
+    "\n",
+    "This is a tutorial to illustrate the main feautures of the Python interface for [libCEED](https://github.com/CEED/libCEED/), the low-level API library for efficient high-order discretization methods developed by the co-design [Center for Efficient Exascale Discretizations](https://ceed.exascaleproject.org/) (CEED) of the [Exascale Computing Project](https://www.exascaleproject.org/) (ECP).\n",
+    "\n",
+    "While libCEED's focus is on high-order finite/spectral element method implementations, the approach is mostly algebraic and thus applicable to other discretizations in factored form, as explained in the [user manual](https://libceed.readthedocs.io/) and [Doxygen documentation](https://codedocs.xyz/CEED/libCEED/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up libCEED for Python\n",
+    "\n",
+    "Install libCEED for Python by running"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! python -m pip install libceed"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## CeedVector\n",
+    "\n",
+    "Here we show some basic examples to illustrate the `CeedVector` class. In libCEED, CeedVectors constitute the main data structure and serve as input/output for `CeedOperator`. \n",
+    "\n",
+    "We illustrate the simple creation of a `CeedVector`, how to specify its size, and how to read or manipulate its data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import libceed\n",
+    "\n",
+    "ceed = libceed.Ceed()\n",
+    "\n",
+    "n = 10\n",
+    "x = ceed.Vector(n)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can visualize the (unset) CeedVector by typing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(x)"
```

Comment:
I don't see the point of printing out uninitialized memory.

Reply:
Ok, I had specified that it was unset, but yes, I agree that it's better to show it after it's set. I have moved it now. Let me know what you think.